models/articleModel.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../db/connection.js');
+
+exports.fetchArticles = (article_id) => {
+    if (isNaN(parseInt(article_id))){
```

Comment:
PSQL is expecting a number when querying a column that holds an integer. This shouldn't be a custom error, you should have an error handler specifically for PSQL error codes. Refer back to the error handling notes on how to achieve this.